### PR TITLE
First push of the ui-project

### PR DESCRIPTION
**Pull request by gunamata**
- Added the antora-ui-default project
- Added a helper file that filters our nav objects specific to the module of the current page

### CODE REVIEW

**Comment by btat** [comment]
@gunamata Could you add the name(s) of the file(s) you've modified so we don't need to look at the files from upstream? It's not obvious since it's all one commit.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+image: node:10.14.2-stretch
```
Comment (btat) [inline .gitlab-ci.yml]: This file should be removed.

File: docs/antora.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+name: antora-ui-default
```
Comment (btat) [inline docs/antora.yml]: We should remove the entire `docs` directory.

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (btat) [inline .vscode/settings.json]: I think this got accidentally checked in and should be removed (and gitignored).

If there are plans for us to use shared settings, we should add this file when it has such settings.

**Review by btat — CHANGES_REQUESTED** [review]

**Comment by gunamata** [comment]
> Could you add the name(s) of the file(s) you've modified so we don't need to look at the files from upstream? It's not obvious since it's all one commit.

Added file `src/helpers/navfilter.js`

File: .gitlab-ci.yml (hunk 1 above)
Comment (gunamata) [reply .gitlab-ci.yml]: Removed

File: docs/antora.yml (hunk 2 above)
Comment (gunamata) [reply docs/antora.yml]: Removed

File: .vscode/settings.json (hunk 3 above)
Comment (gunamata) [reply .vscode/settings.json]: Removed and added to .gitignore

Comment (btat) [reply .vscode/settings.json]: This file is still showing up as of 6b1f8706beab9bfbc949ac6b144b0f21f47d378b.

**Review by btat — APPROVED** [review]

**Pull request closed without merging**